src/interfaces/config.ts

```diff
@@ -6,4 +6,5 @@ export interface IConfig {
   user: string;
   password: string;
   port?: number;
+  tryKeyboard?: boolean;
```

Comment:
I think about moving this property from general `IConfig` to more protocol-specific like `ISftpConfig`. That's because there is no keyboard interactive in FTP protocol.